packages/material-ui/src/Step/Step.test.js

```diff
@@ -1,108 +1,136 @@
 import * as React from 'react';
-import * as PropTypes from 'prop-types';
 import { expect } from 'chai';
 import { getClasses } from '@material-ui/core/test-utils';
 import createMount from 'test/utils/createMount';
 import describeConformance from '../test-utils/describeConformance';
-import { createClientRender, within } from 'test/utils/createClientRender';
+import { createClientRender } from 'test/utils/createClientRender';
 import Step from './Step';
-
-/**
- * Exposes props stringified in the dataset of the `[data-testid="props"]`
- */
-function PropsAsDataset(props) {
-  const elementRef = React.useRef();
-  React.useEffect(() => {
-    const { current: element } = elementRef;
-
-    Object.keys(props).forEach((key) => {
-      // converted to strings internally. writing it out for readability
-      element.dataset[key] = String(props[key]);
-    });
-  });
-
-  return <span data-testid="props" ref={elementRef} />;
-}
-
-/**
- * A component that can be used as a child of `Step`.
- * It passes through all unrelated props to the underlying `div`
- * The props passed from `Step` are intercepted
- */
-function StepChildDiv(props) {
-  const {
-    active,
-    alternativeLabel,
-    completed,
-    disabled,
-    expanded,
-    icon,
-    last,
-    orientation,
-    ...other
-  } = props;
-
-  return <div {...other} />;
-}
-StepChildDiv.propTypes = {
-  active: PropTypes.bool,
-  alternativeLabel: PropTypes.bool,
-  completed: PropTypes.bool,
-  disabled: PropTypes.bool,
-  expanded: PropTypes.bool,
-  icon: PropTypes.node,
-  last: PropTypes.bool,
-  orientation: PropTypes.oneOf(['horizontal', 'vertical']),
-};
+import Stepper from '../Stepper';
+import StepLabel from '../StepLabel';
+import StepButton from '../StepButton';
 
 describe('<Step />', () => {
-  let classes;
+  let stepClasses;
+  let stepButtonClasses;
+  let stepLabelClasses;
   const mount = createMount();
 
   const render = createClientRender();
 
   before(() => {
-    classes = getClasses(<Step />);
+    stepClasses = getClasses(<Step />);
```

Comment:
As the component we are considering

```suggestion
    classes = getClasses(<Step />);
```

Reply:
@oliviertassinari my thought was - if I a test file need classes only for a component which is being tested then I leave `classes` variable otherwise I'd specify CSS class variables more precisely. If you want me to rename it to just `classes` then I should do it for the other updated components as well.
What do you think about my approach?

Reply:
I think that we should keep "classes" to signal that the objective is to focus on one of the component

Reply:
@oliviertassinari Got it! I'll update other components in a couple of minutes

Reply:
Ah, I see you've updated almost all of them already :)